src/v1/user_is_admin.rs

```diff
@@ -33,7 +32,7 @@ impl<'a> Request<'a> {
 }
 
 impl Response {
-    /// Creates a `Response` with the given Synapse and Python versions.
+    /// Creates a `Response` with the given ….
```

Comment:
Uh, this suggestion was not meant literal. There should be a human-readable description of the parameter here, maybe "the given admin flag".